query.go

```diff
@@ -552,11 +564,22 @@ func (qr *QueryResult) Scan(dest ...interface{}) error {
 					return err
 				}
 				*d = b
+			case bool:
```

Comment:
I don't believe `src` can ever be a bool, no? Since rqlite/sqlite does not have such a type. Maybe @otoolep can confirm.

Reply:
As per https://www.sqlite.org/datatype3.html:

> 2.1. Boolean Datatype
> 
> SQLite does not have a separate Boolean storage class. Instead, Boolean values are stored as integers 0 (false) and 1 (true).
> 
> SQLite recognizes the keywords "TRUE" and "FALSE", as of version 3.23.0 (2018-04-02) but those keywords are really just alternative spellings for the integer literals 1 and 0 respectively. 

Therefore rqlite does not have a boolean type.